src/licensedcode/data/licenses/adacore-doc.LICENSE

```diff
@@ -0,0 +1,14 @@
+---
+key: adacore-doc
+short_name: AdaCore Doc License
+name: AdaCore Doc License
+category: Permissive
```

Comment:
`alterations are clearly marked as alterations` category should be Permissive or Free Restricted?

Reply:
That's still permissive